aten/src/ATen/native/Constraints.cpp

```diff
@@ -42,9 +42,9 @@ void sym_constrain_range(
 
     TORCH_CHECK(
       min_val <= size_as_int && size_as_int <= max_val,
-      "Invalid value range for ",
```

Comment:
I think this only exist for the C++ path, not meta kernel.

Reply:
Does that matter? At the meta kernel level they'll only see a symint, which does it matter as much?

Reply:
Oh I was just pointing it out that this assertion behaviour is not symmetric.